src/constants/regex.js

```diff
@@ -6,6 +6,7 @@ export default {
   delegateName: /^[a-z0-9!@$&_.]{3,20}$/,
   transactionId: /^[0-9a-z]{64}/,
   blockId: /^[0-9a-z]{64}/,
+  blockHeight: /[0-9]+/,
```

Comment:
Your changes break the delegate search because the regex you created captures a delegate name that has numbers too.  I think you should change it to

```suggestion
  blockHeight: /^[0-9]+$/,
```
So it only captures numbers and nothing else.

Reply:
@vibratious could you please consider Ali's input and update the PR
- Code changes
- Signing the commits

Thank you so much for your contributions 🙏🏼